content/blog/en/dojo-middleware.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Custom Dojo Middleware
```

Comment:
> Creating reusable functionality in widgets using middleware?

> How to use middleware to create reusable functionality across widgets?

Something like that. I don't really like the term "Custom Dojo Middleware" as is.

Reply:
Yup agreed. I mostly only went through and change the point of view from “I” to “we”.